src/components/InputTodo.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-alert */
+/* eslint-disable react/button-has-type */
```

Comment:
- [x] Linters are tools that perform static analysis on software to detect and report adherence to a set of coding best practices. Instead of disabling eslint, it would be preferable to resolve the issue. :handshake: